warp/backend.go

```diff
@@ -131,100 +121,56 @@ func (b *backend) AddMessage(unsignedMessage *avalancheWarp.UnsignedMessage) err
 		return fmt.Errorf("failed to put warp signature in db: %w", err)
 	}
 
-	var signature [bls.SignatureLen]byte
 	sig, err := b.warpSigner.Sign(unsignedMessage)
 	if err != nil {
```

Comment:
wonder if this function should check sourceChainID and networkID like we check for offchain messages